### DISCUSSION TITLE
No mail when creating account

### DISCUSSION THREAD

**Discussion by JujuLand** [mastodon/mastodon#18445] [General]
Hi,
I have installed Mastodon on my Samsung 5.
I have tried to create an account on mastodon.top, but the mail has never been received.

I first tried with the name Jujuland, and after with Jujuland46, and I never receive the confirmation mail.

Were do I failed ?

Thanks
A+

**Comment by JujuLand** [comment]
I tried again on a new server, and it works.

Sorry for the noise ...

**Reply by Arribaren** [reply]
Hi jujuland 
Est-ce que le mail indiqué était sur le FAI orange ?
Merci.
Jp
